components/ruby/chef-licensing.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "tty-prompt", "~> 0.23"
   spec.add_dependency "faraday", ">= 1", "< 2"
   spec.add_dependency "faraday-http-cache"
-  spec.add_dependency "activesupport", "~> 7.2", ">= 7.2.2.1"
+  spec.add_dependency "moneta", "~> 1.6"
```

Comment:
I understand we are removing activesupport to avoid external dependency issues. But at the same time we are introducing a new dependency that has not received any updates in last 2 years. How can we ensure this receives latest security patches and updates ?